src/cljs/webchange/dashboard/students/subs.cljs

```diff
@@ -20,7 +20,11 @@
 (re-frame/reg-sub
   ::current-student
   (fn [db]
-    (get-in db [:dashboard :current-student])))
+    (let [current-class-id (get-in db [:dashboard :current-class-id])
```

Comment:
access-code is not loaded in lists and also "unassigned students" are stored in a different way.
let's just update the current student on edit.